sample.wdl

```diff
@@ -50,25 +51,29 @@ workflow SampleWorkflow {
                 outputSAM = true,
                 secondaryAlignment = false,
                 howToFindGTAG = howToFindGTAG,
+                addMDtagToSAM = true,
                 dockerImage = dockerImages["minimap2"]
         }
 
-        call transcriptClean.TranscriptClean as transcriptClean {
-            input:
-                SAMfile = minimap2.outputAlignmentFile,
-                referenceGenome = genomeFile,
-                outputPrefix = outputDirectory + "/" + readgroupIdentifier,
-                spliceJunctionAnnotation = spliceJunctionsFile,
-                variantFile = variantVCF,
-                dockerImage = dockerImages["transcriptclean"]
+        if (runTranscriptClean) {
+            call transcriptClean.TranscriptClean as transcriptClean {
+                input:
+                    SAMfile = minimap2.outputAlignmentFile,
+                    referenceGenome = genomeFile,
+                    outputPrefix = outputDirectory + "/" + readgroupIdentifier,
+                    spliceJunctionAnnotation = spliceJunctionsFile,
+                    variantFile = variantVCF,
+                    dockerImage = dockerImages["transcriptclean"]
+            }
         }
     }
 
     output {
+        Array[File] outputSAMsampleWorkflow = if (runTranscriptClean) then select_all(transcriptClean.outputTranscriptCleanSAM) else minimap2.outputAlignmentFile
```

Comment:
This line is too long, see https://biowdl.github.io/styleGuidelines.html#4-line-length-and-line-breaks